docarray/index/backends/hnswlib.py

```diff
@@ -430,7 +430,13 @@ def num_docs(self) -> int:
         if self._num_docs == 0:
             self._num_docs = self._get_num_docs_hnsw()
         return self._num_docs
-
+    def _get_num_docs_hnsw(self) -> int:
+        """
+        Get the number of documents using the HNSW method.
+        """
+        # Access your HNSW index here and retrieve the count of documents
+        hnsw_num_docs = self.hnsw_index.get_document_count()
```

Comment:
I think there is not a `self.hnsw_index` attribute. I think there are multiples inside the `self._hnsw_indices`.

I think u can use the first inside the map to retrieve the count.

Also please make sure to comply with the style contribution standards that you can find in https://github.com/docarray/docarray/blob/main/CONTRIBUTING.md

Also, make sure to signoff your commits so that we can merge your contribution